internal/renderers/femtovg/fonts.rs

```diff
@@ -616,7 +616,7 @@ pub(crate) fn layout_text_lines(
     };
     let mut y = baseline_y;
     let mut start = 0;
-    'lines: while start < string.len() && y + font_height <= max_height {
+    'lines: while start <= string.len() && y + font_height <= max_height {
```

Comment:
This has the effect that for an empty string, the loop will continue at `start == 0` until `y + font_height <= max_height`. In the given test case that's when y is just under 600. That's not right. I think it's correct that the loop is _not_ entered.